python/raft-ann-bench/src/raft-ann-bench/run/__main__.py

```diff
@@ -80,23 +81,24 @@ def run_build_and_search(
     force,
     build,
     search,
+    dry_run,
     k,
     batch_size,
     search_threads,
     mode="throughput",
 ):
     for executable, ann_executable_path, algo in executables_to_run.keys():
         # Need to write temporary configuration
-        temp_conf_filename = f"temporary_{conf_filename}"
-        temp_conf_filepath = os.path.join(conf_filedir, temp_conf_filename)
-        with open(temp_conf_filepath, "w") as f:
+        temp_conf_filename = f"temporary_{conf_filename}_{uuid.uuid1()}.json"
```

Comment:
Let's change this to
```suggestion
        temp_conf_filename = f"{conf_filename}_{algo}_{uuid.uuid1()}.json"
```